get_orgs.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+import requests
+import datetime
+from bs4 import BeautifulSoup
+
+current_date = datetime.datetime.now()
+current_year = current_date.year
+
+query_year = raw_input("Enter a year from 2016 to " + str(current_year) + "\n")
+
+# typecase query_year from string to integer
+query_year = int(query_year)
+
+#Validate input year
+if(query_year < current_year and query_year >= 2016):
```

Comment:
What about having a '<=' here?

Reply:
I understand why you did this. A better way could be checking if the link exists!